unity-renderer/Assets/Features/ScreenshotCamera/Scripts/ScreenshotCamera.cs

```diff
@@ -195,14 +197,24 @@ internal void InstantiateCameraObjects()
             characterCameraTransform = cameraController.GetCamera().transform;
 
             screenshotCamera = Instantiate(cameraPrefab, characterCameraTransform.position, characterCameraTransform.rotation);
+
             screenshotHUDView = Instantiate(screenshotHUDViewPrefab);
 
+            screenshotHUDView.CloseButton?.onClick.AddListener(DisableScreenshotCameraMode);
+            screenshotHUDView.TakeScreenshotButton?.onClick.AddListener(() => CaptureScreenshot(DUMMY_DCL_ACTION_TRIGGER));
```

Comment:
You shouldn't use the Null-conditional operator on unity objects

Reply:
it was only to make tests works, but I can convert it to non-conditional null-check